app/templates/src/_BoilerplateComponent.js

```diff
@@ -1,17 +1,22 @@
 'use strict';
 <% if (templateLanguage === 'Soy') { %>
-import templates from './<%= componentName %>.soy<% if (buildFormat !== 'amd') { %>.js<% } %>';
+import templates from './<%= componentName %>.soy.js';
 import Component from 'metal-component';
 import Soy from 'metal-soy';<% } else if (templateLanguage === 'JSX') { %>
 import JSXComponent from 'metal-jsx';<% } else { %>
-import Component from 'metal-component';<% } %>
+import Component from 'metal-component';<% } %><% if (buildFormat === 'jquery') { %>
+import JQueryAdapter from 'metal-jquery-adapter';<% } %>
+
+import './<%= kebabCaseName %>.scss';
```

Comment:
I'm a little worried about importing the scss by default, since it constricts the bundling process quite a bit. What if someone else wanted to use this component but didn't want to webpack their styles? Just a thought, I'm open to discussion.

@jbalsas , what do you think?